tests/jballerina-unit-test/src/test/resources/test-src/annotations/local_tuple_annotation_test.bal

```diff
@@ -0,0 +1,157 @@
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+type AnnotTupleOne record {|
+   string value;
+|};
+
+type Details record {|
+    string name;
+    int age;
+|};
+ 
+annotation AnnotTupleOne annotOne on type, field;
+annotation AnnotTupleOne annotTwo on type, field;
+annotation Details details on field;
+
+function testAnnotationOnTupleFields() {
+    string k = "chiranS";
+    [@annotOne {value: "10"} int, @annotOne {value: k} int, string...] x1 =  [1, 2, "hello", "world"];
+    map<any> m1 = getLocalTupleAnnotations(typeof x1, "$field$.0");
+    map<any> m2 = getLocalTupleAnnotations(typeof x1, "$field$.1");
+    assertEquality({value: "10"}, <map<anydata>>m1["annotOne"]);
+    assertEquality({value: "chiranS"}, <map<anydata>>m2["annotOne"]);
+}
+
+function testAnnotationOnTupleFields2() {
+    string name = "chiranS";
+    int age = 26;
+    [@details {name: name, age: age} int, @details {name: "name", age: 0} int, string...] x1 =  [1, 2, "hello", "world"];
+    map<any> m1 = getLocalTupleAnnotations(typeof x1, "$field$.0");
+    map<any> m2 = getLocalTupleAnnotations(typeof x1, "$field$.1");
+    assertEquality({name: "chiranS", age: 26},  <map<anydata>>m1["details"]);
+    assertEquality({name: "name", age: 0},  <map<anydata>>m2["details"]);
+}
+
+string gVar = "foo";
+string gVar1 = "bar";
+
+function testAnnotationOnTupleWithGlobalVariable() {
+    [@annotOne {value: gVar} int, int, string...] x1 =  [1, 2, "hello", "world"];
+    map<any> m = getLocalTupleAnnotations(typeof x1, "$field$.0");
+    assertEquality({value: "foo"}, <map<anydata>>m["annotOne"]);
+}
+
+function testMultipleAnnotationsOnLocalTuple() {
+    string k = "chiranS";
+    [@annotOne {value: "foo"} @annotTwo {value: "bar"} int, @details {name: k, age: 0} int, string...] x1 =  [1, 2, "hello", "world"];
+    map<any> m1 = getLocalTupleAnnotations(typeof x1, "$field$.0");
+    map<any> m2 = getLocalTupleAnnotations(typeof x1, "$field$.1");
+    assertEquality({value: "foo"}, <map<anydata>>m1["annotOne"]);
+    assertEquality({value: "bar"}, <map<anydata>>m1["annotTwo"]);
+    assertEquality({name: "chiranS", age: 0},  <map<anydata>>m2["details"]);
+}
+
+function() returns [int] x = function() returns [@annotOne {value: "foo"} int] {return [1];};
+function() returns [int] x2 = function() returns [@annotOne {value: gVar1} @annotTwo {value: gVar2} int] {return [1];};
+function() returns [int, int] x3 = function() returns [@annotOne {value: gVar1} int, @details {name: "name", age: gVar3} int] {return [1, 1];};
+
+function testGlobalAnnotationsOnFunctionPointerReturnType() {
+    map<any> m1 = getLocalTupleAnnotations(typeof x(), "$field$.0");
```

Comment:
Test function name says global but this is local.

Btw we use module-level, not global.

Reply:
Change name to `testTupleAnnotationsOnFunctionPointerReturnType`